examples/simple-service-poc/simple_service.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
+"""
+the requestor agent controlling and interacting with the "simple service"
+"""
+import asyncio
+from datetime import datetime, timedelta, timezone
+import json
+import pathlib
+import queue
+import random
+import string
+import sys
+
+
+
+from yapapi import (
+    NoPaymentAccountError,
+    __version__ as yapapi_version,
+    windows_event_loop_fix,
+)
+from yapapi.executor import Golem
+from yapapi.executor.services import Service
+
+from yapapi.log import enable_default_logger, log_summary, log_event_repr, pluralize  # noqa
+from yapapi.payload import vm
+
+examples_dir = pathlib.Path(__file__).resolve().parent.parent
+sys.path.append(str(examples_dir))
+
+from utils import (
+    build_parser,
+    TEXT_COLOR_CYAN,
+    TEXT_COLOR_DEFAULT,
+    TEXT_COLOR_RED,
+    TEXT_COLOR_YELLOW,
+)
+
+NUM_INSTANCES = 1
+
+
+class SimpleService(Service):
+    STATS_PATH = "/golem/out/stats"
+    PLOT_INFO_PATH = "/golem/out/plot"
+    SIMPLE_SERVICE = "/golem/run/simple_service.py"
+
+    @staticmethod
+    async def get_payload():
+        return await vm.repo(
+        image_hash="8b11df59f84358d47fc6776d0bb7290b0054c15ded2d6f54cf634488",
+        min_mem_gib=0.5,
+        min_storage_gib=2.0,
+    )
+
+    async def start(self):
+        self._ctx.run("/golem/run/simulate_observations_ctl.py", "--start")
```

Comment:
Let's have a string constant for the path here, just like we have for `/golem/run/simple_service.py`, it also appears more than once in the code.

Reply:
done in https://github.com/golemfactory/yapapi/pull/303/commits/44f826e0780c89cea4f1f5aaf2e0f0f9be349569